mobile/android/app/src/main/res/values/styles.xml

```diff
@@ -16,4 +16,68 @@
     <style name="NormalTheme" parent="@android:style/Theme.Light.NoTitleBar">
         <item name="android:windowBackground">@color/whiteColor</item>
     </style>
+
+    <style name="wandegeya">
```

Comment:
We might need to assign another name to this style. `wandegeya` sounds like a place